docs/common/response-handlers.md

```diff
@@ -0,0 +1,51 @@
+A `#!typescript Respond` (declared using `#!typescript {kind: "respond"}`) 
+indicates the desire to handle the response to an Instruction using a
+`#!typescript ResponseHandler`. 
+
+```typescript
+import { 
+    Parameter, 
+    ResponseHandler, 
+    Tags,
+    ParseJson } from "@atomist/rug/operations/Decorators";
+
+import {
+    HandlerContext,
+    HandleResponse,
+    EventPlan,
+    Respondable,
+    Response,
+    ResponseMessage,
+} from "@atomist/rug/operations/Handlers";
+
+@ResponseHandler("GenericSuccessHandler", "Displays a success message in chat")
+@Tags("success")
+class GenericSuccessHandler implements HandleResponse<any> {
+
+    @Parameter({ description: "Success msg", pattern: "@any" })
+    public msg: string;
+
+    public handle(@ParseJson response: Response<any>): EventPlan {
+        const rand = new ChannelAddress("#random");
+        const result = `${this.msg}: ${response.body.status}`;
+        return new EventPlan().add(new DirectedMessage(result, rand));
+    }
+}
+
+```
```

Comment:
Also - how do we import from there? Isn't everything brought in via import relative to the docs/common director?